packages/fx-core/templates/plugins/resource/simpleauth/bicep/input_param.template.bicep

```diff
@@ -0,0 +1,5 @@
+
+param simpleAuth_sku string = 'F1'
+param simpleAuth_serverFarmsName string = '${resourceGroupName}-simpleAuth-serverfarms'
```

Comment:
Change `resourceGroupName` to something like `resourceBaseName`? By the way, we may need to figure out a rule (length, available characters, etc.) for the base name. Can you help on this?

Reply:
Changed to `resourceBaseName`

Reply:
Task added for the task to figure out rule on resourceBaseName.
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10404312